doc/website/getting-started/installation.md

```diff
@@ -64,103 +64,113 @@ QNX SDP 7.0 and 7.1 are supported (shipping with gcc 5.4 and gcc 8.3 respectivel
- | `IOX_MAX_PUBLISHER_HISTORY` | the maximum number chunks available for the publisher history |
- | `IOX_MAX_CHUNKS_ALLOCATED_PER_PUBLISHER_SIMULTANEOUSLY` | the maximum number of chunks a publisher can allocate at a given time |
- | `IOX_MAX_SUBSCRIBERS` | the maximum number of subscribers one `RouDi` instance can manage |
- | `IOX_MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY` | the maximum number of chunks a subscriber can hold at a given time |
- | `IOX_MAX_INTERFACE_NUMBER` | the maximum number for interface ports, which are used for e.g. gateways |
-
-Have a look at [iceoryx_posh_deployment.cmake](../iceoryx_posh/cmake/iceoryx_posh_deployment.cmake) for the default values of this constants.
+Please take a look at the CMake file [build_options.cmake](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_meta/build_options.cmake) to get an overview of the available build options for enabling additional features.
 
 ## Build with script
 
-As an alternative we provide our build-test script which we use to integrate iceoryx into our infrastructure.
-The intention of the script is to more than just building with iceoryx. This is for doing a code coverage scan or for using the adress-sanitizer.
+As an alternative we provide a build-test script which we use to integrate iceoryx into our infrastructure.
+The intention of the script is more than just building with iceoryx. This is for doing a code coverage scan or using the adress-sanitizer.
 The script currently only works for Linux and QNX, it is planned to offer a multi-platform solution.
 
-1. Clone the repository
-
-        git clone https://github.com/eclipse-iceoryx/iceoryx.git
-
-1. Build everything
+ 1. Clone the repository
+    ```bash
+    git clone https://github.com/eclipse-iceoryx/iceoryx.git
+    ```
 
-        cd iceoryx
-        ./tools/iceoryx_build_test.sh build-all
+ 2. Build everything
+    ```bash
+    cd iceoryx
+    ./tools/iceoryx_build_test.sh build-all
+    ```
+    !!! note
+        The build script is installing the header files and binaries into `build/install/prefix`.
 
 You can use the help for getting an overview over the available options:
+    ```bash
+    ./tools/iceoryx_build_test.sh help
+    ```
 
-```bash
-./tools/iceoryx_build_test.sh help
-```
+    !!! tip
+        The examples can be build with `-DEXAMPLES=ON` with iceoryx_meta or `examples` as argument to the script.
 
 ## Build with colcon
 
-Alternatively, iceoryx can be built with [colcon](https://colcon.readthedocs.io/en/released/user/installation.html) to provide a smooth integration for ROS2 developers.
+Alternatively, iceoryx can be built with [colcon](https://colcon.readthedocs.io/en/released/user/installation.html#using-debian-packages) to provide a smooth integration for ROS2 developers.
+For building the iceoryx_integrationtest package is at least a minimal ROS2 installation needed (https://docs.ros.org/en/foxy/Installation/Linux-Install-Debians.html).
```

Comment:
To build the iceoryx_integrationtest package one requires a minimal ROS2 installation.